source/parser/parser.h

```diff
@@ -755,7 +755,7 @@ class Parser : public SceneTask
         void Parse_Rel_Term (EXPRESS& Express, int *Terms);
 
         /// Parses a REL_TERM comparing two strings.
-        void Parse_Rel_String_Term (const UCS2 *lhs, EXPRESS& Express, int Terms);
+        void Parse_Rel_String_Term (const UCS2 *lhs, EXPRESS& Express, int *Terms);
```

Comment:
From what I see, the new version of this function always sets `*Terms=1`, and only ever sets `Express[0]`; wouldn't it make more sense to give the function a full overhaul and change its signature to

    double Parse_Rel_String_Term (const UCS2 *lhs);

and leave all the worry about vector promotion up to the caller entirely?